streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -380,16 +380,23 @@
 
     /** {@code default.windowed.key.serde.inner} */
     @SuppressWarnings("WeakerAccess")
+    @Deprecated
     public static final String DEFAULT_WINDOWED_KEY_SERDE_INNER_CLASS = "default.windowed.key.serde.inner";
     private static final String DEFAULT_WINDOWED_KEY_SERDE_INNER_CLASS_DOC = "Default serializer / deserializer for the inner class of a windowed key. Must implement the " +
         "<code>org.apache.kafka.common.serialization.Serde</code> interface.";
 
     /** {@code default.windowed.value.serde.inner} */
     @SuppressWarnings("WeakerAccess")
+    @Deprecated
     public static final String DEFAULT_WINDOWED_VALUE_SERDE_INNER_CLASS = "default.windowed.value.serde.inner";
     private static final String DEFAULT_WINDOWED_VALUE_SERDE_INNER_CLASS_DOC = "Default serializer / deserializer for the inner class of a windowed value. Must implement the " +
         "<code>org.apache.kafka.common.serialization.Serde</code> interface.";
 
+    public static final String WINDOW_INNER_CLASS_DESERIALISER = "window.inner.class.deserializer";
```

Comment:
I know this is the name we voted on in the KIP, but I wonder if it should be `WINDOWED` instead of `WINDOW`? I honestly didn't notice it before, but "windowed" seems more consistent with the actual Serde name. 
If you agree, you can just update the KIP and send a quick note to the KIP thread on the mailing list to clarify the name change